test/jdk/java/text/Format/common/Bug6215962.java

```diff
@@ -55,25 +55,25 @@ static void testMessageFormat() {
         mf2 = new MessageFormat("{0}", Locale.JAPAN);
         check(mf1, mf2, false);
 
-        mf1 = new MessageFormat("{0}", new Locale("ja", "JP"));
+        mf1 = new MessageFormat("{0}", Locale.of("ja", "JP"));
```

Comment:
Locale.JAPAN?

Reply:
The test intends to compare the generated `MessageFormat` objects (`mf2` & `mf1`) from using a constant `Locale.JAPAN` and the factory method. So I believe leaving it as `Locale.of()` makes sense.